src/Umbraco.Web/Mvc/UmbracoAuthorizeAttribute.cs

```diff
@@ -55,7 +56,7 @@ public UmbracoAuthorizeAttribute(bool redirectToUmbracoLogin)
         {
             if (redirectToUmbracoLogin)
             {
-                _redirectUrl = Current.Configs.Global().GetBackOfficePath(Current.HostingEnvironment).EnsureStartsWith("~");
+                _redirectUrl = new GlobalSettings().GetBackOfficePath(Current.HostingEnvironment).EnsureStartsWith("~");
```

Comment:
This is part of `Umbraco.Web`. Most of this class is already migrated (e.g the constructor). So this will never be executed and is mostly just to allow tests to pass